<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图书管理系统</string>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bookButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/books.png</normaloff>:/icons/books.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="themeButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/dashboard.png</normaloff>:/icons/dashboard.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="userButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/person.png</normaloff>:/icons/person.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="settingButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="dayButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/today.png</normaloff>:/icons/today.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>731</width>
      <height>601</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>书籍管理</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>741</width>
        <height>571</height>
       </rect>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>所有图书</string>
       </attribute>
       <widget class="QTableWidget" name="book_table">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>711</width>
          <height>531</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>书号</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>书名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>书籍介绍</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>书籍种类</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>书籍作者</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>出版社</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>价格</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>添加书籍</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>书名：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="add_book_name">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>50</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>简介：</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="add_book_des">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>90</y>
          <width>301</width>
          <height>271</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>50</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>书号：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="add_book_code">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>50</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>100</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>作者：</string>
        </property>
       </widget>
       <widget class="QComboBox" name="add_book_author">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>90</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="add_book_type">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>140</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>150</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>类别：</string>
        </property>
       </widget>
       <widget class="QComboBox" name="add_book_publisher">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>190</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>200</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>出版社：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="add_book_price">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>250</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>250</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>价格：</string>
        </property>
       </widget>
       <widget class="QPushButton" name="add_book_search">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>50</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>搜索</string>
        </property>
       </widget>
       <widget class="QPushButton" name="add_book_save_Button">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>430</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>编辑书籍</string>
       </attribute>
       <widget class="QComboBox" name="edit_book_author">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>100</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="edit_book_save_Button">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>440</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>书名：</string>
        </property>
       </widget>
       <widget class="QPushButton" name="edit_book_search">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>60</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>搜索</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>260</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>价格：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="edit_book_name">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>60</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>110</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>作者：</string>
        </property>
       </widget>
       <widget class="QComboBox" name="edit_book_publisher">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>200</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="edit_book_price">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>260</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QTextEdit" name="edit_book_des">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>100</y>
          <width>301</width>
          <height>271</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="edit_book_type">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>150</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>210</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>出版社：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>简介：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="edit_book_code">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>60</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>60</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>书号：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>160</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>类别：</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>设置管理</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>741</width>
        <height>571</height>
       </rect>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>类别</string>
       </attribute>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>50</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>类别：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="add_category_name">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>50</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="add_category_Button">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>40</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="category_table">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>661</width>
          <height>321</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>类别</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>作者</string>
       </attribute>
       <widget class="QPushButton" name="add_author_Button">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>70</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>80</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>作者：</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="author_table">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>130</y>
          <width>661</width>
          <height>311</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>作者</string>
         </property>
        </column>
       </widget>
       <widget class="QLineEdit" name="add_author_name">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>80</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>出版社</string>
       </attribute>
       <widget class="QPushButton" name="add_publisher_Button">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>60</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>60</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>出版社：</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="publisher_table">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>661</width>
          <height>311</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>出版社</string>
         </property>
        </column>
       </widget>
       <widget class="QLineEdit" name="add_publisher_name">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>60</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="theme_groupBox">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>161</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(216, 216, 216);</string>
    </property>
    <property name="title">
     <string>主题设置</string>
    </property>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>换个主题，换个心情！</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dark_orange_button">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>主题一</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dark_blue_button">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>主题二</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dark_gray_button">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>主题三</string>
     </property>
    </widget>
    <widget class="QPushButton" name="qdark_button">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>360</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>主题四</string>
     </property>
    </widget>
    <widget class="QPushButton" name="theme_change_Button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>260</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>》</string>
     </property>
    </widget>
   </widget>
   <zorder>tabWidget</zorder>
   <zorder>bookButton</zorder>
   <zorder>themeButton</zorder>
   <zorder>userButton</zorder>
   <zorder>settingButton</zorder>
   <zorder>dayButton</zorder>
   <zorder>theme_groupBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
